### ISSUE TITLE
Can not access module settings

### ISSUE THREAD

**Issue by jeff9600** [PrestaShop/autoupgrade#294]
**Describe the bug**
Hello,
I want to update my version 1.6.1.8 of Prestashop in version 1.6.1.23. To carry out my update, I installed the latest version of 1-Click Upgrade as requested by the Prestashop documentation.
Unfortunately, when I click on the button allowing me to go into the module settings, the following error message appears: "Fatal error: Uncaught Error: Call to undefined method AdminSelfUpgrade :: displayErrors () in / home / clients / 839385930dcd51377b6711bec9d0d0f8 / temp / adminlheuCN / functions.php: 281 Stack trace: # 0 /home/clients/839385930dcd51377b6711bec9d0d0f8/temp/adminlheuCN/functions.php(422): checkingTab ('AdminSelfUpgrad ...') # 1 / home / clients / 839385930dcd51377b6711bec9d0d0f8 / temp / classes / Dispatcher.php (347): runAdminTab ('AdminSelfUpgrad ...', false) # 2 /home/clients/839385930dcd51377b6711bec9d0d0f8/temp/adminlheuCN/index.php(58): DispatcherCore-> dispatch () # 3 {main} thrown in /home/clients/839385930dcd51377b6711bec9d0d0f8/temp/adminlheuCN/functions.php on line 281 "

**Upgrade environment (please complete the following information):**
 - Version you started from: 1.6.1.8

**Displayed log**
I do not have access

Insert upgrade / rollback log here
```

**Expected behavior**
I would like you to tell me where the problem comes from and what I need to do to fix it.

**Screenshots**
![erreur-module-1-click-upgrade](https://user-images.githubusercontent.com/49283787/55570387-a407be00-5703-11e9-9712-acf315429b87.jpg)


**Server (please complete the following information):**
 - Web Server: I don't know
 - PHP Version: [e.g. 7.0]
 - Module Version [e.g. 4.6.0]

**Additional context**
...

**Comment by Quetzacoalt91** [comment]
Hi there,

This error is triggered when the core tries to tell you that you cannot access the configuration page (You've not been granted access to its configuration page). While we fix the error, you should ask the shop administrator to give you access to the configuration page of the module.

**Comment by Quetzacoalt91** [comment]
This issue will be fixed on the next version thanks to #302
